### ISSUE TITLE
See a list of all people/sites that received donations

### ISSUE THREAD

**Issue by bitcoinuser** [Donate4Fun/donate4fun#19]
I would like to see a list of all people/sites that received donations from donate4fun

Obs:

1- The cards will show the total received and if the user verified or no the account on donat4fun
2- The total amount will consider too the donations sent directly to lightning address using donate4fun extension.
3- The list will include too the sites that have v4v.
4- The list could include too a list of sites that never received donations using donate4fun but have v4v.

**Comment by nikicat** [comment]
1-3 is pretty easy and may be done within #22 
4 requires a web scraper for the entire web, so is a pretty hard task, is it what you mean?

**Comment by bitcoinuser** [comment]
> 4 requires a web scraper for the entire web, so is a pretty hard task, is it what you mean?

Yes, but if is very hard task, I think have more important things to work first.
